ggshield/pre_receive_cmd.py

```diff
@@ -163,6 +173,7 @@ def prereceive_cmd(ctx: click.Context, web: bool, prereceive_args: List[str]) ->
 'git push -o breakglass'"""
                 )
             return return_code
-
+    except TimeoutError:
```

Comment:
Don't we want to add `TimeoutError` in our scan exceptions handler?

Reply:
We catch this exception because it is explicitly raised by ExitAfter [here](https://github.com/GitGuardian/ggshield/blob/agateau/150/error-handling/ggshield/pre_receive_cmd.py#L52). I don't understand what you mean, where should it also be caught?

Reply:
After discussing this, we agreed to leave it as is, because this TimeoutError is specific to this part of the code (it is raised by ExitAfter)